--- kicad-2013.04.10/pcb_calculator/dialogs/pcb_calculator_frame_base.fbp.orig	2013-04-15 23:36:30.000000000 -0500
+++ kicad-2013.04.10/pcb_calculator/dialogs/pcb_calculator_frame_base.fbp	2013-04-20 10:57:12.474799647 -0500
@@ -472,7 +472,7 @@
                                                             <property name="aui_row"></property>
                                                             <property name="best_size"></property>
                                                             <property name="bg"></property>
-                                                            <property name="bitmap">Load From File; F:\kicad-launchpad\testing\pcb_calculator\bitmaps\regul_3pins.xpm</property>
+                                                            <property name="bitmap">Load From File; ../bitmaps/regul_3pins.xpm</property>
                                                             <property name="caption"></property>
                                                             <property name="caption_visible">1</property>
                                                             <property name="center_pane">0</property>
--- kicad-2013.04.19/pcb_calculator/dialogs/pcb_calculator_frame_base.cpp.orig	2013-04-20 19:48:35.000000000 -0500
+++ kicad-2013.04.19/pcb_calculator/dialogs/pcb_calculator_frame_base.cpp	2013-04-20 20:55:35.626238874 -0500
@@ -9,7 +9,7 @@
 
 #include "pcb_calculator_frame_base.h"
 
-#include "../../../../testing/pcb_calculator/bitmaps/regul_3pins.xpm"
+#include "../bitmaps/regul_3pins.xpm"
 #include "../bitmaps/arrow_bottom.xpm"
 #include "../bitmaps/arrow_top.xpm"
 #include "../bitmaps/color_code_multiplier.xpm"
